packages/manager/apps/pci-workflow/src/api/hooks/order.ts

```diff
@@ -19,7 +27,7 @@ export const useProjectAddons = (projectId: string, ovhSubsidiary: string) => {
   });
 };
 
-export const useInstanceSnapshotPricing = (
+export const findInstanceSnapshotPricing = (
```

Comment:
The `useInstanceSnapshotPricing` still exists at the bottom of the file, but its content has been split into two sub functions `findInstanceSnapshotPricing` & `findLocalZoneInstanceSnapshotPricing`, because there are two ways of finding this price

Reply:
@y4nnL 
Yes, I see the point 👍🏻
But you cannot invoke any hook inside a regular JS function, that violates the rules of hooks:
https://legacy.reactjs.org/docs/hooks-rules.html#only-call-hooks-from-react-functions